src/pygeoprocessing/geoprocessing.py

```diff
@@ -1629,9 +1640,18 @@ def reproject_vector(
 
     # Get the SR of the original_layer to use in transforming
     base_sr = layer.GetSpatialRef()
-
     # Create a coordinate transformation
-    coord_trans = osr.CoordinateTransformation(base_sr, target_sr)
+    # In GDAL 3.0 spatial references no longer ignore Geographic CRS 
+    # Axis Order and conform to Lat first, Lon Second. Transforms
+    # expect (lat, lon) order as opposed to the GIS friendly (lon, lat). 
+    # See https://trac.osgeo.org/gdal/wiki/rfc73_proj6_wkt2_srsbarn Axis order
+    # issues. SetAxisMappingStrategy(osr.OAMS_TRADITIONAL_GIS_ORDER) swaps 
+    # the axis order, which will use Lon,Lat order for Geographic CRS, 
+    # but otherwise leaves Projected CRS alone. 
```

Comment:
I think this comment isn't relevant here anymore since we pass in the axis mapping strategy as a parameter, but I think it may be helpful at the top of the module where you define the default. Would you mind moving it to the single default instance at the top?

Reply:
Done in [c9f0d6](https://github.com/natcap/pygeoprocessing/pull/31/commits/c9f0d66d3513cb8559367280e605b008d244f30c)